proposals/csharp-11.0/low-level-struct-improvements.md

```diff
@@ -332,10 +332,11 @@ Note: An expression whose type is not a `ref struct` type always has a *safe-to-
 #### Rules for method invocation
 <a name="rules-method-invocation"></a>
 
-The span safety rules for method invocation will be updated in several ways. The first is by recognizing the impact that `scoped` has on arguments. For a given argument `a` that is passed to parameter `p`:
+The span safety rules for method invocation will be updated in several ways. The first is by recognizing the impact that `scoped` has on arguments. For a given argument `expr` that is passed to parameter `p`:
```

Comment:
fwiw, I thought `a` was a useful name because it implied `argument` and had a duality with `p`. Either way is fine.